kernelci/build.py

```diff
@@ -754,6 +754,12 @@ def install_kernel(kdir, tree_name, tree_url, git_branch, git_commit=None,
                 kimages.append(name)
                 image_path = os.path.join(root, name)
                 shutil.copy(image_path, install_path)
+    for files in os.listdir(output_path):
+        for name in kimage_names:
+            if name == files:
+                kimages.append(name)
+                image_path = os.path.join(output_path, name)
+                shutil.copy(image_path, install_path)
```

Comment:
Isn't this already done by the block of code just above?

Reply:
Oh I see, `vmlinux` is not placed in the boot directory.  I think it would be better to change the loop above to iterate over the 2 paths instead of duplicating it.

Reply:
no, it seek only in arch/$arch/boot

Reply:
The first loop is a find, mine is just a "ls". I dont see how to combine both cleanly

Reply:
OK I'll post a suggestion to improve this or a follow-up PR.